model/src/main/kotlin/ResolvedConfiguration.kt

```diff
@@ -41,6 +42,12 @@ data class ResolvedConfiguration(
      */
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val packageCurations: List<ResolvedPackageCurations> = emptyList(),
+
+    /**
+     * All resolutions that apply to this [OrtResult].
+     */
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Not strictly related to this commit, but if we continue to add default values to the properties, we could swap the `@JsonInclude(JsonInclude.Include.*)` annotations on the properties with a `@JsonInclude(JsonInclude.Include.NON_DEFAULT)` annotation on the class.

Reply:
> Jackson needs to create an instance of the class using the no-argument constructor

That's indeed crazy. Do you have a reference for that?

Reply:
It's in the [docs](https://fasterxml.github.io/jackson-annotations/javadoc/2.7/com/fasterxml/jackson/annotation/JsonInclude.Include.html#NON_DEFAULT):
> When used for a POJO, definition is that only values that differ from the default values of POJO properties are included. This is done by creating an instance of POJO using zero-argument constructor, and accessing property values: value is used as the default value by using equals() method, except for the case where property has `null` value in which straight null check is used.